### ISSUE TITLE
/rnn/char_lstm notebook example is broken

### ISSUE THREAD

**Issue by recastrodiaz** [dmlc/mxnet#1116]
I found a few issues while trying to run this notebook:

- the learning rate is too big, leading to NRR=Infinity and Prep=Infinity
- the optimizer was changed from 'sgd' to 'rmsprop', but the hyperparameters were not updated: there is no 'momentum' hyperparameter in rmsprop.
- the 'input.txt' file is not automatically downloaded when running the ./get_ptb_data.sh script
- the script get_ptb_data.sh script should use [#!/usr/bin/env](http://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my)

I'll create a PR to fix the issues above.

**Comment by recastrodiaz** [comment]
PR available here: https://github.com/dmlc/mxnet/pull/1117

**Comment by mli** [comment]
merged, many thanks!
